### PR TITLE
Win bash completion

### PR DESCRIPTION
**Pull request by melund**
This fixed bash completion for me. It failed because files had spaces in their names.

### CODE REVIEW

File: xonsh/completer.py [hunk 1]
```diff
@@ -448,10 +448,10 @@ def _collect_completions_sources():
                  builtins.__xonsh_env__.get('BASH_COMPLETIONS', ()))
         for path in paths:
             if path.is_file():
-                sources.append('source ' + str(path))
+                sources.append('source "{}"'.format(path.as_posix()))
             elif path.is_dir():
                 for _file in (x for x in path.glob('*') if x.is_file()):
-                    sources.append('source ' + str(_file))
+                    sources.append('source "{}"'.format(_file.as_posix))
```
Comment (scopatz) [inline xonsh/completer.py]: doesn't this need to be `_file.as_posix()`?

File: xonsh/completer.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def _path_from_partial_string(inp, pos=None):
 
 COMPLETION_SKIP_TOKENS = {'man', 'sudo', 'time', 'timeit', 'which'}
 
-BASH_COMPLETE_SCRIPT = """source {filename}
+BASH_COMPLETE_SCRIPT = """source "{filename}"
```
Comment (scopatz) [inline xonsh/completer.py]: Yep, definite oversight.  Mea culpa.

**Comment by scopatz** [comment]
Thanks a million @melund! Just the one method call issue.

**Comment by melund** [comment]
Done

**Comment by scopatz** [comment]
thanks!

**Pull request merged**